lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:cboard_mobile/Settings/ThemeManager.dart';
+import 'package:cboard_mobile/Settings/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'routes.dart';
+
+class Setting_wrapper extends StatelessWidget {
+  // ThemeNotifier theme = new ThemeNotifier();
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: Color(0xFF23036A),
+        accentColor: Color(0xFFE3CAFF),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        scaffoldBackgroundColor: Colors.white,
+      ),
+      home: Scaffold(
+        backgroundColor: Color(0xffE5E5E5),
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Settings'),
+        ),
+        body: MyStatefulWidget(),
+      ),
+      routes: getRoute(),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  final sections = ['People', 'Language', 'System', 'Help'];
+  final List<Map<String, List<Object>>> features = <Map<String, List<Object>>>[
```

Comment:
Do you think we can/should combine these you objects to create one larger object? Maybe something like this might look more efficient:

```
'People': [
  'Language': [XYZ, ...],
  'Speech': [XYZ, ...]
]

Reply:
yup. Done!